compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -18,6 +18,7 @@
 package org.wso2.ballerinalang.compiler.semantics.analyzer;
 
 import io.ballerina.identifier.Utils;
+import io.ballerina.runtime.api.flags.SymbolFlags;
```

Comment:
Why are we importing a runtime class? Please fix.

Reply:
Changed